cmd/operator/run.go

```diff
@@ -45,8 +43,7 @@ func main() {
 		fmt.Printf("Config path: %s", configPath)
 		return
 	}
-	fmt.Println("WARNING: Running protoform with defaults, no config file sent.")
-	runProtoform("")
+	panic("config file not provided")
```

Comment:
@msenmurugan  Won't this change break OLM deployments?
https://github.com/blackducksoftware/community-operators/blob/master/upstream-community-operators/synopsys/synopsys-operator.2019.4.0.clusterserviceversion.yaml#L323

Reply:
done